ssvsigner/ekm/remote_key_manager.go

```diff
@@ -104,10 +105,31 @@ func (km *RemoteKeyManager) AddShare(
 		PubKey:           pubKey,
 	}
 
-	if err := km.signerClient.AddValidators(ctx, shareKeys); err != nil {
+	statuses, err := km.signerClient.AddValidators(ctx, shareKeys)
+	if err != nil {
 		return fmt.Errorf("add validator: %w", err)
 	}
 
+	for _, status := range statuses {
+		switch status {
+		case web3signer.StatusImported:
+			continue
```

Comment:
I think we should cover this part in tests

Reply:
@y0sher I think `continue` looks clearer than nothing, but I can remove it